point_viewer_grpc/src/lib.rs

```diff
@@ -138,6 +138,14 @@ impl OctreeDataProvider for GrpcOctreeDataProvider {
     }
 }
 
-pub fn octree_from_address(addr: &str) -> Result<Octree> {
-    Octree::from_data_provider(Box::new(GrpcOctreeDataProvider::from_address(addr)?))
+pub fn octree_from_grpc_address(addr: &str) -> Result<Box<Octree>> {
+    let prefix = "grpc://";
+    if !addr.starts_with(prefix) {
+        return Err(format!("Invalid grpc address: it has to start with {}.", prefix).into());
+    }
+    let addr_no_prefix = &addr[prefix.len()..].to_string();
```

Comment:
Can we change `&addr[prefix.len()..].to_string()` to `addr[prefix.len()..]`?

Reply:
this causes "doesn't have a size known at compile-time", but the following works
` let addr_no_prefix: &str = &addr[prefix.len()..];`